deployment/keystone/ocr3config.go

```diff
@@ -300,7 +298,7 @@ func (r configureOCR3Request) generateOCR3Config() (OCR2OracleConfig, error) {
 
 type configureOCR3Response struct {
 	ocrConfig OCR2OracleConfig
-	proposal  *timelock.MCMSWithTimelockProposal
+	ops       *timelock.BatchChainOperation
```

Comment:
match the updated style that lib func return ops and changesets create proposals from ops